src/video_core/renderer_opengl/gl_rasterizer.cpp

```diff
@@ -622,6 +622,16 @@ void RasterizerOpenGL::SyncViewport() {
             }
             flags[Dirty::Viewport0 + index] = false;
 
+            if (!regs.viewport_transform_enabled) {
+                GLfloat x = static_cast<float>(regs.render_area.x);
```

Comment:
```suggestion
                const auto x = static_cast<GLfloat>(regs.render_area.x);
```

Ditto for the ones below